backend/src/main/java/com/team4/backend/util/JwtUtil.java

```diff
@@ -35,12 +36,19 @@ public void init() {
     }
 
     public String generateToken(User user) {
-        Map<String, Role> claims = new HashMap<>();
+        Map<String, String> claims = new HashMap<>();
         Long expirationTimeLong = Long.parseLong(expirationTime);
         final Date createdDate = new Date();
         final Date expirationDate = new Date(createdDate.getTime() + expirationTimeLong * 1000);
 
-        claims.put("role", user.getRole());
+        claims.put("role", user.getRole().toString());
+        claims.put("firstName", user.getFirstName());
+        claims.put("lastName", user.getLastName());
+        claims.put("phoneNumber", user.getPhoneNumber());
+
+        if (user.getRole().equals(Role.MONITOR))
+            claims.put("companyName", ((Monitor) user).getCompanyName());
```

Comment:
generate token should always return the same thing regardless of Role, I think a better solution is needed.
Call me on discord when you read this.